packages/telemetry/src/index.ts

```diff
@@ -150,6 +137,15 @@ export class AstroTelemetry {
 			context.anonymousId = `CI.${meta.ciName || 'UNKNOWN'}`;
 		}
 
+		if (this.debug.enabled) {
```

Comment:
Moved this down here so that it would output all data collected, not just the specific event payload.